WireUI/Sources/WireDesign/Placeholder.swift

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
Placeholder of what ?

Reply:
A target in a Swift package needs at least a file.